README.md

```diff
@@ -1,29 +1,78 @@
 
+1. You fill out the form on the 'New Newsletter' page and click submit.
 
-Most of the web is about exposing data stored in applications.
+2. Information about you, your newsletter content and any additional information such as media items are all sent to the application server.
 
-(feel free to choose a more RESTful example for example domain)
+3. The server interprets the information, recognizes that the request is for a new newsletter and it generates the new record in the database and performs a myriad of background tasks (updates the newsletter counter, possibly sends notification emails, et al)
 
-http verbs mean something urls mean something
+4. Next the server sends a response back to the client, this does not necessarily mean that the newsletter was posted, the response could be that there was an error posting or something like that. However in this case we will say that the newsletter post went through properly, so the server sends a success message and tells the browser what page to go to and render. 
 
-put it all together and we get predictable guessable urls that describe the things they are representing. we call these things a resource.
+5. Lastly the browser receives the server information and gives the user feedback, in this case it shows you a message saying that your newsletter was successfully posted.
 
-/posts vs /pictures
 
-the 7 restful routes 7 restful actions
+## RESTful Conventions in Rails
 
-REST is a concept - not a technology. for right now, just think of it as standard URLS and HTTP verbs that describe what they are trying to do
+Rails has RESTful principles built into its core, so whether you are utilizing the built in view rending system or using the application purely as API you will have the tools necessary to follow standardized routing procedures.
 
-diagram that shows the mirroring of CRUD, to ROutes to controller Actions
+If you were to build out a feature in Rails that entails all of the potential RESTful routes, you would have seven different RESTful URLs, HTTP verbs (more on those later), and actions. As an example, imagine that you have a Newsletter module in your application, if you run ```rake routes``` in the terminal you will receive the following output:
+
+```
+newsletters     GET     /newsletters(.:format)          newsletters#index
+                POST    /newsletters(.:format)          newsletters#create
+new_newsletter  GET     /newsletters/new(.:format)      newsletters#new
+edit_newsletter GET     /newsletters/:id/edit(.:format) newsletters#edit
+newsletter      GET     /newsletters/:id(.:format)      newsletters#show
+                PATCH   /newsletters/:id(.:format)      newsletters#update
+                PUT     /newsletters/:id(.:format)      newsletters#update
+                DELETE  /newsletters/:id(.:format)      newsletters#destroy
+```
```

Comment:
The issue is that they are going to type `rake routes` in this folder and expect that output. Also there is extra stuff there (that left column and the .format stuff) that will confuse them